pkg/ovs/configjob.go

```diff
@@ -0,0 +1,120 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+	"github.com/openstack-k8s-operators/ovs-operator/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ovnclient "github.com/openstack-k8s-operators/ovn-operator/api/v1beta1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ConfigJob - prepare job to configure ovn-controller
+func ConfigJob(
+	ctx context.Context,
+	h *helper.Helper,
+	k8sClient client.Client,
+	instance *v1beta1.OVS,
+	labels map[string]string,
+) ([]*batchv1.Job, error) {
+
+	var jobs []*batchv1.Job
+	runAsUser := int64(0)
+	privileged := true
+	jobTTLAfterFinished := int32(10)
```

Comment:
why 10 and not some other value? (zero or whatever) Please add a comment explaining the rationale.

Reply:
actually I don't know, I probably set it like that during development and forgot to change it. Now it's set to 0 so finished jobs will be cleaned automatically right after they will finish.